tests/fixture-valid.js

```diff
@@ -128,31 +138,30 @@ function checkChannel(channel) {
     }
   }
 
-  let usedAsCoarseOnlyChannel = true;
-  if (channel.fineChannelAliases.length > 0) {
-    for (const alias of channel.fineChannelAliases) {
-      if (definedChannelKeys.includes(alias)) {
-        result.errors.push(`Fine channel alias '${alias}' in channel '${channel.key}' is already defined.`);
-      }
-      definedChannelKeys.push(alias);
-    }
-
-    usedAsCoarseOnlyChannel = fixture.modes.some(
-      mode => channel.getFinenessInMode(mode) < channel.maxFineness
+  // Fine channels
+  for (const alias of channel.fineChannelAliases) {
+    checkChannelKey(alias);
```

Comment:
No, the schema allows either ChannelKey or TemplateChannelKey (extending Channel to overwrite the possible value for fineChannelAliases would be overkill) – it's mainly a visual purpose. With `checkChannelKey`, I really check if the key is named correctly.